README.md

```diff
@@ -63,29 +63,7 @@ snippet shows an example of how to scan using pre-built detectors.
 
 ####  Sample Code
 
-```java
-// By default, the client reads the API key from the environment variable NIGHTFALL_API_KEY
-try (NightfallClient c = NightfallClient.Builder.defaultClient()) {
-
-    // Define some detectors to use to scan your data
-    Detector creditCard = new Detector("CREDIT_CARD_NUMBER");
-    creditCard.setMinConfidence(Confidence.LIKELY);
-    creditCard.setMinNumFindings(1);
-    Detector ssn = new Detector("US_SOCIAL_SECURITY_NUMBER");
-    ssn.setMinConfidence(Confidence.POSSIBLE);
-    ssn.setMinNumFindings(1);
-
-    // A rule contains a set of detectors to scan with
-    DetectionRule rule = new DetectionRule(Arrays.asList(creditCard, ssn), LogicalOp.ANY);
-
-    List<String> payload = Arrays.asList("hello world", "my SSN is 678-99-8212", "4242-4242-4242-4242");
-    ScanTextConfig config = ScanTextConfig.fromDetectionRuleUUIDs(Arrays.asList(rule), 20);
-    ScanTextRequest req = new ScanTextRequest(payload, config);
-
-    ScanTextResponse response = c.scanText(req);
-    System.out.println("findings: " + response.getFindings());
-}
-```
+see [the TextScannerExample class](src/main/java/ai/nightfall/examples/TextScannerExample.java)
```

Comment:
maybe also link the FileScanner example, or just link the examples directory collectively?

Reply:
the file scanner is also linked on line 86, does that work?

Reply:
oof, i can't read.